exercises/practice/difference-of-squares/.meta/lib/example.dart

```diff
@@ -9,5 +9,5 @@ class DifferenceOfSquares {
 
   num _sum(int input) => input * (input + 1) ~/ 2;
 
-  List<int> _range(int length) => new List<int>.generate(length, (i) => i + 1);
+  List<num> _range(int length) => new List<num>.generate(length, (i) => i + 1);
```

Comment:
Why is that necessary?

Reply:
The dart linter applied this change. Let me look up the lint rule for it.

Reply:
Hmm, well I can't reproduce the output with `dart analyze`, `dart format`, `dart_style`, or `dart fix`.

Maybe it was my IDE? Anyhow, I'll revert it